### PR TITLE
add acrn-dm -emul parameter list

### PR DESCRIPTION
**Pull request by liuhang-bit**
add acrn-dm -emul parameter list in the end of acrn-dm paramter charpter.

### CODE REVIEW

**Review by gvancuts — CHANGES_REQUESTED** [review]
Thanks for kicking this off @liuhang-bit 

I have not done a complete and thorough review yet, but there are a couple of high-level comments on this:
* Can you add all parameters that each of these can take?
* Can you be more verbose for some, e.g. `virtio-hdcp` meaning it's a `Virtio hdcp` device isn't bringing a lot of information to the user, how do they use it, etc.
* Capitalization is important in the documentation, so I expect to see `HDCP` instead of `hdcp` for example

**Comment by NanlinXie** [comment]
Closed due to duplicated with #6445.

File: doc/user-guides/acrn-dm-parameters.rst [hunk 1]
```diff
@@ -472,3 +472,92 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
...
+----
+
+.. list-table:: Emul Items List For PCI Device Type
+   :header-rows: 1
+
+   * - Emul Type
+     - Description
+
+   * - "xhci"
+     - Usb controller used to support USB3.0 devices, provide downward compatibility
```
[the remark this answers is not in the document]
Comment (liuhang-bit) [reply doc/user-guides/acrn-dm-parameters.rst:485]: yes

File: doc/user-guides/acrn-dm-parameters.rst [hunk 2]
```diff
@@ -472,3 +472,92 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
...
+     - Windows graphic driver requires this virtualized LPC device to complete the display function
+
+   * - "ivshmem"
+     - Virtualized PCI device used specifically for shared memory between VMs
+
+   * - "ahci"
+     - Advanced Host Controller Interface used to connect with SATA storage device
+
+   * - "ahci-cd"
+     - Advanced Host Controller Interface used to connect with AT Attachment Packet Interface device
```
[the remark this answers is not in the document]
Comment (liuhang-bit) [reply doc/user-guides/acrn-dm-parameters.rst:500]: CD-Rom is one of AT Attachment Packet Interface type device,i will add it as an example

File: doc/user-guides/acrn-dm-parameters.rst [hunk 3]
```diff
@@ -472,3 +472,92 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
...
+   * - "igd-lpc"
+     - Windows graphic driver requires this virtualized LPC device to complete the display function
+
+   * - "ivshmem"
+     - Virtualized PCI device used specifically for shared memory between VMs
+
+   * - "ahci"
+     - Advanced Host Controller Interface used to connect with SATA storage device
+
+   * - "ahci-cd"
```
[the remark this answers is not in the document]
Comment (liuhang-bit) [reply doc/user-guides/acrn-dm-parameters.rst:499]: It seems like ahci-hd and ahci is the same in the code

File: doc/user-guides/acrn-dm-parameters.rst [hunk 4]
```diff
@@ -472,3 +472,92 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
...
+   * - "amd_hostbridge"
+     - Virtualized PCI AMD hostbridge
+
+   * - "hostbridge"
+     - Virtualized PCI hostbridge
+
+   * - "pci_gvt"
+     - Using Intel GVT-g technology
+
+   * - "virtio-blk"
```
[the remark this answers is not in the document]
Comment (liuhang-bit) [reply doc/user-guides/acrn-dm-parameters.rst:511]: they are all the acrn-dm  -emul parameter ,the example of how to use -emul is in the doc of device model parameter

File: doc/user-guides/acrn-dm-parameters.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (gvancuts) [reply doc/user-guides/acrn-dm-parameters.rst:485]: Thanks, can you make this more explicit in the text.

File: doc/user-guides/acrn-dm-parameters.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (gvancuts) [reply doc/user-guides/acrn-dm-parameters.rst:500]: Thanks

File: doc/user-guides/acrn-dm-parameters.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (gvancuts) [reply doc/user-guides/acrn-dm-parameters.rst:499]: I agree, can we add it to the list and indicate it's an alias?

File: doc/user-guides/acrn-dm-parameters.rst (hunk 4 above)
[the remark this answers is not in the document]
Comment (gvancuts) [reply doc/user-guides/acrn-dm-parameters.rst:511]: Not really, what I'm asking to have included here are all the options that `virtio-blk` can take. In the case of `virtio-blk` these are listed in the HLD (see [here](https://projectacrn.github.io/latest/developer-guides/hld/virtio-blk.html#usage)) but 1) I'm not sure these are still accurate and 2) we should make those explicitly part of this guide (or at a minimum reference some other place in our doc where to find them).

We should do this (i.e. detail all options) for **all** the `emul` types in this doc.

File: doc/user-guides/acrn-dm-parameters.rst [hunk 5]
```diff
@@ -472,3 +472,92 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
...
+     - Virtio block type device
+
+   * - "virtio-coreu"
+     - Used for PAVP session management
```
[the remark this answers is not in the document]
Comment (liuhang-bit) [reply doc/user-guides/acrn-dm-parameters.rst:515]: protected audio and visual path used for user vm to protect their audio and visual content

**Pull request closed without merging**